srcpkgs/libsearpc/patches/01-glib-include-kludge.patch

```diff
@@ -0,0 +1,26 @@
+  Seafile sources sometimes include searpc-client.h in an extern "C" block.
+  It then includes glib.h and glib-object.h, and recent versions of glib don't
+  work when included within extern "C".
+
+  This is the easy fix: wrap the include of glib headers in extern "C++" to undo
+  the extern "C".
+
+  Upstream is aware of the problem, and should have a fix soon, at which time this
+  kludge won't be needed anymore.
```

Comment:
please point upstream's issue with a link if available, much more concise.
juste like `upstream issue: http://<the-url>`